INSTALL.rst

```diff
@@ -258,3 +258,21 @@ above. The 'return URL' of the B2ACCESS configuration in this case can be set
 to ``http://localhost:5000/api/oauth/authorized/b2access/``
 
 If working on the web UI, see also: https://github.com/EUDAT-B2SHARE/b2share/wiki/Developer's-corner.
+
```

Comment:
This addition conflicts with the removal of all docs you did in https://github.com/EUDAT-B2SHARE/b2share/pull/1535. Can you move it into the documentation repository?